packages/language-support/src/tests/syntaxValidation/syntacticValidation.test.ts

```diff
@@ -675,6 +683,41 @@ describe('Syntactic validation spec', () => {
     ]);
   });
 
+  test.fails(
+    'Syntax validation errors on multiline unfinished property keys',
+    () => {
+      const query = `RETURN {\`something
+    foo
+    
+    bar: "hello"}`;
```

Comment:
The error for this case has worsened and it would require some work in the mono-repo to fix it